source/libs/utils.ts

```diff
@@ -52,7 +52,7 @@ export const groupBy = (iterable: Iterable<string>, grouper: (item: string) => s
 // Concats arrays but does so like a zipper instead of appending them
 // [[0, 1, 2], [0, 1]] => [0, 0, 1, 1, 2]
 // Like lodash.zip
-export const flatZip = (table: string[][], limit = Infinity) => {
+export const flatZip = (table: any[][], limit = Infinity) => {
```

Comment:
Use `unknown` instead of `any` whenever possible.

Reply:
This function should ideally be generic. Like: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e95a4f0f1fc68e2289fbe5cc3405787fa6b2b8a1/types/lodash/common/array.d.ts#L3302-L3331